frontend/src/components/layout/header/UserControls.vue

```diff
@@ -14,7 +14,7 @@
             </svg>
         </VBtn>
         <VToolbarTitle class="hidden-sm-and-down username mr-6">
-            Hello, <span>{{ user.name }}</span>
+            Hello, <span>{{ user().name }}</span>
```

Comment:
No need to do like this, original issue is that mapGetter should be placed in `computed` attr, and not `methods` attr inside component definition